.github/workflows/tests_gpu_cpp.yml

```diff
@@ -30,16 +24,14 @@ env:
   CI_CUDA_ARCH: 86
   COVERAGE_FLAGS: "--cov=pennylane_lightning --cov-report=term-missing --cov-report=xml:./coverage.xml --no-flaky-report -p no:warnings --tb=native"
   GCC_VERSION: 11
-  TF_VERSION: 2.10.0
-  TORCH_VERSION: 1.11.0+cpu
 
 concurrency:
-  group: tests_lgpu_cpp-${{ github.ref }}-${{ github.event_name }}-${{ inputs.lightning-version }}-${{ inputs.pennylane-version }}
+  group: tests_gpu_cpp-${{ github.ref }}-${{ github.event_name }}-${{ inputs.lightning-version }}-${{ inputs.pennylane-version }}
   cancel-in-progress: true
 
 jobs:
   builddeps:
-    if: github.event.pull_request.draft == false
+    if: ${{ github.event.pull_request.draft == false && contains(github.event.pull_request.labels.*.name, 'ci:use-gpu-runner') }}
```

Comment:
Hmm.. are these actions still triggered on push-to-master? What do you think about the ideal behaviour on push-to-master?

Reply:
For now, we can set it with an OR boolean gate. The only caveat is that the GPU runners will run in draft mode if we add the label. Having a set of conditions where the label only operates when in a PR will require some more thought.